packages/insomnia-app/app/ui/components/panes/grpc-request-pane/__tests__/use-action-handlers.test.ts

```diff
@@ -0,0 +1,78 @@
+import { createBuilder } from '@develohpanda/fluent-builder';
+import { renderHook } from '@testing-library/react-hooks';
+import { ipcRenderer } from 'electron';
+import { mocked } from 'ts-jest/utils';
+
+import { GrpcRequestEventEnum } from '../../../../../common/ipc-events';
+import { grpcIpcMessageParamsSchema } from '../../../../../network/grpc/__schemas__/grpc-ipc-message-params-schema';
+import { grpcIpcRequestParamsSchema } from '../../../../../network/grpc/__schemas__/grpc-ipc-request-params-schema';
+import { GrpcMethodTypeEnum } from '../../../../../network/grpc/method';
+import { prepareGrpcMessage as _prepareGrpcMessage, prepareGrpcRequest as _prepareGrpcRequest } from '../../../../../network/grpc/prepare';
+import { GrpcDispatch } from '../../../../context/grpc';
+import { GrpcActionTypeEnum } from '../../../../context/grpc/grpc-actions';
+import useActionHandlers from '../use-action-handlers';
+
+jest.mock('electron');
+jest.mock('../../../../../network/grpc/prepare', () => {
+  const module = {
+    prepareGrpcMessage: jest.fn(),
+    prepareGrpcRequest: jest.fn(),
+  };
+
+  return module;
+});
+
+const mockGrpcDispatch = mocked<GrpcDispatch>(jest.fn());
+const prepareGrpcMessage = mocked(_prepareGrpcMessage);
+const prepareGrpcRequest = mocked(_prepareGrpcRequest);
+
+const messageParamsBuilder = createBuilder(grpcIpcMessageParamsSchema);
+const requestParamsBuilder = createBuilder(grpcIpcRequestParamsSchema);
+
+describe('useActionHandlers', () => {
+  it('should send start', async () => {
+    const requestParams = requestParamsBuilder.build();
+
+    prepareGrpcRequest.mockResolvedValue(requestParams);
+
+    const { result } = renderHook(() => useActionHandlers(requestParams.request._id, 'env', GrpcMethodTypeEnum.unary, mockGrpcDispatch));
```

Comment:
```suggestion
    const { result } = renderHook(() => useActionHandlers(
      requestParams.request._id,
      'env',
      GrpcMethodTypeEnum.unary,
      mockGrpcDispatch
    ));
```

One suggestion was going to make was that it get's hard to read out there this long, but, ultimately, the real feedback would be to give `useActionHandlers` named arguments (i.e. an object).  That's clearly out of scope for this PR so I'll just mention it and maybe we can update in the future.

Reply:
A linter should identify when something should be moved onto new lines and automatically fix. I agree, in a future PR we could (and should) use an option bag